src/layouts/BaseLayout/BaseLayout.jsx

```diff
@@ -49,9 +49,7 @@ function BaseLayout({ children }) {
               {isFullExperience ? (
                 children
               ) : (
-                <Suspense fallback={null}>
-                  <TermsOfService />
-                </Suspense>
```

Comment:
So ... are we just getting rid of the terms and service page?

Reply:
for this iteration, yah

Reply:
Just opening this back up. Can you bring back the ToS page, and introduce your own flag for the onboarding page. The ToS page is still on the books to be done so we will still need it. This will also require some changes to `useUserAccessGate` so that if they haven't completed ToS they do that then can go to the sync provider (soon^tm) and then finally to onboarding.

Reply:
so i said this feature falls under TOS initiative, which means when we bring back TOS page, if the flag is off we should not render both TOS and select org for users